server/api/room.ts

```diff
@@ -119,6 +121,8 @@ const createRoom: RequestHandler<
 	OttResponseBody<OttApiResponseRoomCreate>,
 	OttApiRequestRoomCreate
 > = async (req, res) => {
+	const validatedBody = createRoomSchema.parse(req.body);
```

Comment:
Also `validatedBody` is currently not used. You should replace usages of `req.body` with the validated body object.

nit: rename `validatedBody` to `body`

Reply:
Ah yeah I had it like that before but wasn't sure if it was the same. I'll do that.